### DISCUSSION TITLE
iOS & Android Native App

### DISCUSSION THREAD

**Discussion by UnixPhonez** [stoatchat/discussions#333] [Feature Suggestions]
An iOS and Native App would be nice and an android app that doesn't use the browser as the one on f-droid does.

**Comment by phoebehfharris** [comment]
There's a semi-official Android client in the works, but I don't know about iOS
[Maintainer edit: This project is longer affiliated with us. See https://github.com/stoatchat/for-android for our official Android app]

**Reply by roke-julian-lockhart** [reply]
Regarding iOS, @UnixPhonez and @phoebehfharris, see [`discussions/804#discussioncomment-10885998`](https://github.com/stoatchat/discussions/discussions/804#discussioncomment-10885998) [^1] (although be aware of [`discussions/791#discussioncomment-10883646`](https://github.com/stoatchat/discussions/discussions/791#discussioncomment-10883646)).

[^1]: [`discussions/674#discussioncomment-14973316`](https://github.com/stoatchat/discussions/discussions/674#discussioncomment-14973316)

**Comment by BoardingRaichu** [comment]
This one is major for me. I spend 99% of my time with computer like devices, on my iPad. Having an iPadOS version would be amazing!
